### ISSUE TITLE
Bug: Labeler and Issue Templates Using Wrong Labels

### ISSUE THREAD

**Issue by leahcimto** [theforumhelpers/theforumhelpers.github.io#155]
**Describe the bug**
The labeler and issue templates are using the wrong labels as we recently changed them.

**To Reproduce**
1. make an issue/pr
2. notice that if you make an issue there will be no labels and if you make a pr it will make the labels.

**Expected behavior**
The issues and prs should be marked with the new labels

**Browser Information**
N/A

**Additional context**
None

**Comment by leahcimto** [comment]
I looked over labeler.yml and was wondering if this would work:

```yml
type: Forum Helpers List:
  - forumhelpers/curators.json
  - forumhelpers/managers.json
```
As type has a colon after it. Does anyone know?

**Comment by Accio1** [comment]
Related: Issue templates have to be updated as well in order to add automatic labels.

**Comment by Accio1** [comment]
> I looked over labeler.yml and was wondering if this would work:
> 
> ```yaml
> type: Forum Helpers List:
>   - forumhelpers/curators.json
>   - forumhelpers/managers.json
> ```
> 
> As type has a colon after it. Does anyone know?

I'm not sure. I can do a bit of research on the repo for the actions labeller to see if labels with colons in them will work.

**Comment by leahcimto** [comment]
After testing on a [test repo](https://github.com/leahcimto/a-test-repository) it seems like it does not work with labels with colons.

**Comment by leahcimto** [comment]
Scratch that I'm testing one more time

**Comment by leahcimto** [comment]
You know what, github actions is hard to work with I'm not going to test this lol

**Comment by Shluffy** [comment]
@Accio1 is this still needs triage?

**Comment by Accio1** [comment]
@Shluffy generally labels on closed issues don't really matter.

**Comment by Shluffy** [comment]
> @Shluffy generally labels on closed issues don't really matter.

@Accio1 no, it genarally doesn't. this is to point out about what i was saying on on your profile
